src/test/scala/org/camunda/feel/impl/builtin/BuiltinNumberFunctionTest.scala

```diff
@@ -180,6 +180,18 @@ class BuiltinNumberFunctionsTest
     eval(" abs(-10) ") should be(ValNumber(10))
   }
 
+  "abs() function" should "be invoked with named parameter number" in {
+
+    eval(" abs(number: 1) ") should be(ValNumber(1))
+    eval(" abs(number: -1) ") should be(ValNumber(1))
+  }
+
+  "abs() function" should "be invoked with named parameter n" in {
```

Comment:
We can use the `it` statement to group the test cases.

```suggestion
  it should "be invoked with named parameter n" in {
```